apps/nova/src/components/Plans/data.json

```diff
@@ -0,0 +1,112 @@
+{
+  "data": {
+    "monthly": [
+      {
+        "id": 1,
+        "title": "begining",
+        "description": "start english learning by fixed steps with 60 minutes a week and develop your skills flexiblely with specific tutor",
```

Comment:
Let's write the title and the description in Arabic. Becuase each plan has the same title/description across all subscription periods, we can remove it from this file and write it only once in the plans file and use `intl`.